src/main/kotlin/stickyWallet/sql/tables/AccountsTable.kt

```diff
@@ -0,0 +1,13 @@
+package stickyWallet.sql.tables
+
+import org.jetbrains.exposed.sql.Table
+import stickyWallet.configs.PluginConfiguration
+
+object AccountsTable : Table(
+    PluginConfiguration.StorageSettings.storageTablePrefix + "_accounts"
+) {
+    val playerName = varchar("player_name", 255)
+    val playerUUID = varchar("player_uuid", 255)
```

Comment:
Could be more specific with field lengths here, if i'm not wrong usernames are max 16 chars and uuids are max 36

Reply:
_Yes_, but it doesn't affect it, it just says the max size CAN be 255. Forward compatibility 😄